docs/analyzer.md

```diff
@@ -6,13 +6,10 @@ bugs at runtime.
 
 ## Installation
 
-The analyzer works in Visual Studio 2015 Update 1 or later. It can be installed
-[from NuGet](https://www.nuget.org/packages/FakeItEasy.Analyzer) for each project
-that needs it:
-
-```
-PM> Install-Package FakeItEasy.Analyzer
-```

```

Comment:
I removed the part showing how to install the package. By now people should be used to installing NuGet packages, they don't need to be told. And anyway, I doubt many people still use the package manager console (the only place where this command would work).